src/proto/google/protobuf/any.ts

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable */
```

Comment:
Is this also an automatically added file?

Reply:
Yes, When executing `protocgen.sh`, `any.ts` is generated.